proposals/simd/NewOpcodes.md

```diff
@@ -77,37 +75,60 @@
+| i8x16.add                     | 0x6e   | i16x8.add                 | 0x8e   | i32x4.add                 | 0xae   | i64x2.add                 | 0xce   |
+| i8x16.add_sat_s               | 0x6f   | i16x8.add_sat_s           | 0x8f   | ---- add_sat ----         | 0xaf   | -------------             | 0xcf   |
+| i8x16.add_sat_u               | 0x70   | i16x8.add_sat_u           | 0x90   | ---- add_sat ----         | 0xb0   | -------------             | 0xd0   |
+| i8x16.sub                     | 0x71   | i16x8.sub                 | 0x91   | i32x4.sub                 | 0xb1   | i64x2.sub                 | 0xd1   |
+| i8x16.sub_sat_s               | 0x72   | i16x8.sub_sat_s           | 0x92   | ---- sub_sat ----         | 0xb2   | -------------             | 0xd2   |
+| i8x16.sub_sat_u               | 0x73   | i16x8.sub_sat_u           | 0x93   | ---- sub_sat ----         | 0xb3   | -------------             | 0xd3   |
+| f64x2.ceil                    | 0x74   | f64x2.nearest             | 0x94   | -------------             | 0xb4   | -------------             | 0xd4   |
+| f64x2.floor                   | 0x75   | i16x8.mul                 | 0x95   | i32x4.mul                 | 0xb5   | i64x2.mul                 | 0xd5   |
+| i8x16.min_s                   | 0x76   | i16x8.min_s               | 0x96   | i32x4.min_s               | 0xb6   | i64x2.eq                  | 0xd6   |
+| i8x16.min_u                   | 0x77   | i16x8.min_u               | 0x97   | i32x4.min_u               | 0xb7   | i64x2.ne                  | 0xd7   |
+| i8x16.max_s                   | 0x78   | i16x8.max_s               | 0x98   | i32x4.max_s               | 0xb8   | i64x2.lt_s                | 0xd8   |
+| i8x16.max_u                   | 0x79   | i16x8.max_u               | 0x99   | i32x4.max_u               | 0xb9   | i64x2.gt_s                | 0xd9   |
+| f64x2.trunc                   | 0x7a   |                           | 0x9a   | i32x4.dot_i16x8_s         | 0xba   | i64x2.le_s                | 0xda   |
+| i8x16.avgr_u                  | 0x7b   | i16x8.avgr_u              | 0x9b   | ---- avgr_u ----          | 0xbb   | i64x2.ge_s                | 0xdb   |
+| i16x8.extadd_pairwise_i8x16_s | 0x7c   | i16x8.extmul_low_i8x16_s  | 0x9c   | i32x4.extmul_low_i16x8_s  | 0xbc   | i64x2.extmul_low_i32x4_s  | 0xdc   |
+| i16x8.extadd_pairwise_i8x16_u | 0x7d   | i16x8.extmul_high_i8x16_s | 0x9d   | i32x4.extmul_high_i16x8_s | 0xbd   | i64x2.extmul_high_i32x4_s | 0xdd   |
+| i32x4.extadd_pairwise_i16x8_s | 0x7e   | i16x8.extmul_low_i8x16_u  | 0x9e   | i32x4.extmul_low_i16x8_u  | 0xbe   | i64x2.extmul_low_i32x4_u  | 0xde   |
+| i32x4.extadd_pairwise_i16x8_u | 0x7f   | i16x8.extmul_high_i8x16_u | 0x9f   | i32x4.extmul_high_i16x8_u | 0xbf   | i64x2.extmul_high_i32x4_u | 0xdf   |
 
 | f32x4 Op        | opcode | f64x2 Op        | opcode |
 | --------------- | ------ | --------------- | ------ |
```

Comment:
(I can't seem to add a comment to the line I want, so commenting here instead).
It is a bit arbitrary, but we could put f32x4.ceil and f64x2.ceil here, then we can fit f64x2.{floor,trunc,nearest} into `0xd2-0xd4`, that might make the LLVM opcode definitions a bit simpler? Since the 64x2 opcodes will all be a constant number of opcodes away from 32x4.

Reply:
Where exactly were you thinking about putting the `ceil`s? When I was splitting up these operations, I was trying to optimize for filling holes below 0x80 to save code size. For a group of just a few related operations like this, the difference in simplicity in the LLVM definitions is very minor.

Reply:
inside 0xe2, 0xee. Oh okay, then optimizing for < 0x80 sg, thanks!